var/spack/repos/builtin/packages/openradioss/package.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import inspect
+import os
+from shutil import move
+
+from spack.package import *
+
+
+class Openradioss(CMakePackage):
+    """Altair Radioss is an industry-proven analysis solution that helps users evaluate and
+    optimize product performance for highly nonlinear problems under dynamic loadings. For more
+    than 30 years, organizations have used Altair Radioss to streamline and optimize the digital
+    design process, replace costly physical tests with quick and efficient simulation, and speed
+    up design optimization iterations – all so users and organizations can improve product quality,
+    reduce costs, and shorten development cycles."""
+
+    homepage = "https://www.openradioss.org/"
+    url = "https://github.com/OpenRadioss/OpenRadioss/archive/refs/tags/latest-20220914.tar.gz"
+
+    version("20220914", sha256="8e160707dacd729c72ebeb7c6ef4daa10d71859c444f00810c4896dc2f6edd0a")
```

Comment:
I didn't get the provided sha256 value for that version, instead:

```suggestion
    version("20220914", sha256="7764c3bbaaee63d78938ea632cde2b90fa56c9ad954aca184b392ae2b76dae16")
```

Reply:
`spack checksum` also provided the following for the latest version 😁 :

```
    version("20220915", sha256="9f3913366165d54cea1b5307f2b37fbb24ec49d2f3d84b39b5f3e3ff2a8f7f82")
```